docker/brokers/docker-compose.yml

```diff
@@ -1,22 +1,15 @@
 # This file configures COCOS brokers. 
 # MESSAGE_BROKER can be either 'nats' or 'rabbitmq'.
 #
-# Each broker has a unique profile for configuration. The available profiles are:
-# - nats: Uses 'nats' as MESSAGE_BROKER.
-# - rabbitmq: Uses 'rabbitmq' as MESSAGE_BROKER.
-#
 
 services:
   broker:
```

Comment:
Is this being imported somewhere? I am not seeing it

Reply:
https://github.com/ultravioletrs/cocos-ai/blob/7b31e2181828e21c6d2a00eb20dc7c6b35dd4f2a/docker/docker-compose.yml#L12